### ISSUE TITLE
Flash Message not autohiding

### ISSUE THREAD

**Issue by Adoyle2014** [haojia321/flash-message-toast#1]
The flash message appears and looks nice but it doesn't autohide.  After the default 2 second delay, it just wobbles like a new message does.  Setting autoHide to true instead of relying on the default makes no difference.  Setting autoHide to false does not show the close button.  I am using the meyerweb-reset package and am not controlling any loading of packages if that helps.  Also my fourseven:scss package is only at 3.13.0 which is the latest compatible version for my other packages.

Looks good though and is easy to implement.
Here is the code i am testing it out in:
```import{FlashMessage} from 'meteor/haojia321:flash-message-toast';
Template.createUser.events({
    'submit form':function(evt) {
        evt.preventDefault();
        var email = evt.target.email.value,
            password = evt.target.password.value,
            confirmPassword = evt.target.confirmPassword.value,
            firstName = evt.target.firstName.value,
            lastName = evt.target.lastName.value;

        if(password === confirmPassword) {
            Accounts.createUser({
                email: email,
                password: password,
                firstName: firstName,
                lastName: lastName
            })
        } else {
            FlashMessage.show({
                align: 'topRight',
                text: 'Passwords do not match',
                type: 'error',
                delay: 2000,
                autoHide: false
            })
        }


    }
});```

**Comment by haojia321** [comment]
Parameter should be hideDelay not delay.
You need to showCloseButton=true to make the close button appear.

I tried 
`      FlashMessage.show({   
            align: 'topRight',   
            text: 'Passwords do not match',   
            type: 'error',   
            hideDelay: 7000,
            autoHide: true   
        });`

It hides after 7 seconds.

**Comment by Adoyle2014** [comment]
OK, this is my function: 
`myFlashMessage = function(message, type) {
    FlashMessage.show({
        align: 'topCenter',
        text: message,
        type: type,
        hideDelay: 2000,
        autoHide: true
    })
};`
And i call it:
`myFlashMessage(err.reason, 'error');`

I get the flash message and after 2 seconds, it looks like another message drops down and then it stays there.  I get the same result if i call FlashMessage directly instead of the myFlashMessage function call.
Any ideas

**Comment by haojia321** [comment]
I tested below code in a new Template:
Template.test.onRendered(function() {
    myFlashMessage = function(message, type) { FlashMessage.show({ align: 'topCenter', text: message, type: type, hideDelay: 2000, autoHide: true }) };
    myFlashMessage('test flash message', 'error');
});

It is working fine. Can you also try above code please?
Which browser are you using?

**Comment by Adoyle2014** [comment]
I am using chrome and i tested it in firefox also.  Same result. Here are the packages i am using in case you see a conflict or that i am missing something.
meyerweb-reset
cwaring:modernizr
materialize:materialize
meteorhacks:flow-layout
haojia321:flash-message-toast
accounts-password
accounts-base
kadira:flow-router
kadira:blaze-layout
meteorhacks:fast-render
fourseven:scss
aldeed:collection2
alanning:roles
momentjs:moment
arillo:flow-router-helpers

and also the usual included packages that come with meteor.
I am using METEOR@1.4.3.1

**Comment by haojia321** [comment]
Did you try the code I posted above? Im think it might because of the place(in a callback) you call flashmessage.

**Comment by haojia321** [comment]
Can you also create a fresh meteor project and try to reproduce the issue? If you still can reproduce it in new project, can you post that new project in here so that I can take a look?

**Comment by haojia321** [comment]
@Adoyle2014 any luck? have you tried the code I pasted above?

**Comment by Adoyle2014** [comment]
Hi, I have tried the code in my current project and cannot get it to work correctly.  I started a fresh project with my original code and it works.  I then added all of my packages into the new project and it still worked. I can't find the reason why it won't work in my current project so i am moving on to a different flash message package but will use this one in other materialize projects since it looks better.  If i find out what i was doing to have it not work, I will let you know. Thanks for the help.
